spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/DynamicConformanceJob.scala

```diff
@@ -55,6 +56,10 @@ object DynamicConformanceJob {
   private val menasBaseUrls = MenasConnectionStringParser.parse(conf.getString("menas.rest.uri"))
 
   def main(args: Array[String]) {
+    // This should be the first thing the app does to make secure Kafka work with our CA.
+    // After Spring activates JavaX, it will be too late.
+    SecureKafka.setSecureKafkaProperties(conf)
```

Comment:
I just want to be explicit of what we are doing.

Reply:
It could be actually in a class that both `StandardizationJob`  and `DynamicConformanceJob` can extend. This would move us toward shared code base of the two applications.

Reply:
Yeah, we can do this as part of the task of combining Standardization and Conformance codebase.